### ISSUE TITLE
add - phylogenetic independent contrast

### ISSUE THREAD

**Issue by caseywdunn** [caseywdunn/evowikipedia#2]
Add a page for phylogenetic independent contrasts.

**Comment by mwpennell** [comment]
There is a brief description of the method in the Phylogenetic Comparative Methods page https://en.wikipedia.org/wiki/Phylogenetic_comparative_methods

Though this should definitely have its own page

**Comment by eliotmiller** [comment]
So, provide a link to PICs from the PCM page, pull out the current PIC info and expand?

**Comment by mwpennell** [comment]
yeah i'm working on the PCM page at the moment, so don't please make the
PIC page independently and then cross-link when I'm done first round of
edits. will be fast. doing it piecemeal

**Comment by eliotmiller** [comment]
No worries, I can't officially work on the project until 3:30 to 5 PM eastern North America time today. Will see where issue is at then.
